hydrolib/core/models.py

```diff
@@ -150,3 +153,11 @@ def _get_serializer(cls) -> Callable:
     @classmethod
     def _get_parser(cls) -> Callable:
         return DIMRParser.parse
+
+    @validator("component", pre=True)
+    def validate_component(cls, v):
+        return to_list(v)
+
+    @validator("coupler", pre=True)
```

Comment:
You can combine this one into the previous validator by giving it multiple fields.